src/main/java/edu/indiana/dlib/amppd/controller/AmpUserController.java

```diff
@@ -110,5 +112,12 @@
 		log.info(" Fetched Email id for a token using resetPasswordGetEmail():"+res.getEmailid());
 		return res;
 	  }
+	  
+	  @GetMapping(path="/account/getUser/{Id}")
+	  public @ResponseBody AmpUser getUser(@PathVariable Long Id) {
```

Comment:
As we discussed, and following the convention,, the path can simply be path="/account/{Id}", since this is a simple get request to return the entity object. Similarly, if we are doing an update on user account, it would be POST "/account/{Id}"